flutter_app/lib/changelog/ChangelogService.dart

```diff
@@ -0,0 +1,11 @@
+class ChangelogService {
```

Comment:
Eventually will use real services and injection, but still exploring the package to use for that in flutter